runtime/List.res

```diff
@@ -480,19 +478,22 @@ let rec fromArrayAux = (a, i, res) =>
 
 let fromArray = a => fromArrayAux(a, Array.length(a) - 1, list{})
 
-let toArray = (x: t<_>) => {
+let toArray = (x: list<_>) => {
   let len = length(x)
   let arr = A.makeUninitializedUnsafe(len)
   fillAux(arr, 0, x)
   arr
 }
 
-let toShuffled = xs => {
+let shuffle = xs => {
```

Comment:
Not sure about this one.

On the one hand, it may be confusing that in the `Array` module `shuffle` mutates whereas in the `List` module it does not. On the other hand, we know that `list` is immutable...

Reply:
yeah I'm a bit torn on this one too.

Reply:
what should we do about this one @cknitt?

Reply:
Hmm... I guess either is fine with me. Any preference from your side @zth?

Reply:
my personal preference would be to use shuffle given list is immutable and the signature is clear, but up to you guys!